### PR TITLE
docs: translate zh/resources

### PR DESCRIPTION
**Pull request by WingLim**
- [x] faq.md
- [x] comparison.md

### CODE REVIEW

**Review by linbuxiao — APPROVED** [review]
非常棒！翻译这篇的工作量是巨大的，但对于细节能看出你仍然考究。

File: site/docs/resources/comparison.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In other words, it does not use any code of competing projects, but it will stil
 
 ::: tip Choose your programming language first
 Given that you're reading the docs of a framework within the JavaScript ecosystem, you are likely looking for something to run on Node.js (or Deno).
-However, if that's not you, scroll down for a comparison of what programming languages are suited for bot development.
+However, if that's not you, [scroll down](#comparison-with-frameworks-in-other-programming-languages) for a comparison of what programming languages are suited for bot development.
```
Comment (KnorpelSenf) [inline site/docs/resources/comparison.md:14]: Good idea!

**Comment by WingLim** [comment]
Wait for #79 merged

**Comment by KnorpelSenf** [comment]
Can this be merged now?

**Pull request merged**